localization/pose_initializer/schema/pose_initializer.schema.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Pose Initializer Node",
+  "type": "object",
+  "definitions": {
+    "pose_initializer": {
+      "type": "object",
+      "properties": {
+        "gnss_pose_timeout": {
+          "type": "number",
+          "description": "The duration that the GNSS pose is valid. [sec]",
+          "default": "3.0",
+          "minimum": 0.0
+        },
+        "stop_check_enabled": {
+          "type": "boolean",
```

Comment:
Technically, this variable is a boolean when substituted, but since it is a string in this file, please change it to a string. This is because JSON schema validation will not pass.

Please treat other variables in the same manner.
![image](https://github.com/autowarefoundation/autoware.universe/assets/24854875/fed2c543-4f4b-4558-b285-0c15688e6c65)

Reply:
Oh.. okay!